pom.xml

```
@@ -358,7 +364,7 @@
                     <appServicePlanName>java-functions-app-service-plan</appServicePlanName>
                     <region>westus</region>
                     <runtime>
-                        <os>mac</os>
+                        <os>windows</os>
```

```suggestion
                        <os>linux</os>
```
Our function apps are running on Linux, at this point I'd make everything uniform :)

---

Fixed in https://github.com/pagopa/pagopa-ecommerce-transactions-functions/pull/5/commits/15c774203e3d09be5454f564708e428b353a7eed